scripts/us_census/pep/monthly_population/.pylintrc

```diff
@@ -0,0 +1,340 @@
+[MESSAGES CONTROL]
```

Comment:
Was this file copied from another folder? We should need this per directory as we use `--style google` for all py files.

Reply:
Yes It is copied to see the settings. We will remove this file in our scripts folder.